content/docs/introduction/faq.md

```diff
@@ -11,15 +11,24 @@ toc: full-width
 ### What is Prometheus?
 
 Prometheus is an open-source systems monitoring and alerting toolkit
-with an active ecosystem. See the [overview](/docs/introduction/overview/).
+with an active ecosystem.
+It is the only system directly supported by [Kubernetes](https://kubernetes.io/) and the de facto standard across the [cloud native ecosystem](https://landscape.cncf.io/).
+See the [overview](/docs/introduction/overview/).
 
 ### How does Prometheus compare against other monitoring systems?
 
 See the [comparison](/docs/introduction/comparison/) page.
 
 ### What dependencies does Prometheus have?
 
-The main Prometheus server runs standalone and has no external dependencies.
+The main Prometheus server runs standalone as a single monolithic binary and has no external dependencies.
```

Comment:
This is not correct, we ship other files with prometheus such as the consoles

Reply:
IMO it's correct enough, the console templates are super optional and I count them more as something you can let Prometheus serve than being part of Prometheus or required by it.

Reply:
I would tend to agree with Julius, which is why I proposed the change. I am also fine not having it, see below.